text_generation/causal_lm/cpp/CMakeLists.txt

```diff
@@ -5,7 +5,6 @@ cmake_minimum_required(VERSION 3.15)
 project(causal_lm)
 
 add_subdirectory(../../../thirdparty/openvino_tokenizers/ "${CMAKE_CURRENT_BINARY_DIR}/openvino_tokenizers/")
-
```

Comment:
Put the line back :) It splits general settings and particular executable settings